discovery-provider/alembic/trigger_sql/ddl.sql

```diff
@@ -141,3 +141,12 @@ COMMIT;
 BEGIN;
     create index if not exists tracks_ai_attribution_user_id on tracks (ai_attribution_user_id, is_current) where is_current = true and ai_attribution_user_id is not null;
 COMMIT;
+
+
+-- 5/4/23: strip newline for cid
+BEGIN;
+  UPDATE "tracks"
+  SET "track_cid" = regexp_replace(trim("track_cid"), E'\\s+', ' ', 'g')
+  WHERE is_current = true AND LENGTH(track_cid) != 46;
```

Comment:
wait actually can you change this to where it starts with "Qm" and is 47 chars? otherwise in the future it would include v2 CIDs since they're != 46 chars (not a big deal either I guess since it just removes the newline)

Reply:
oh yea gotcha